content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -60,33 +63,33 @@ provisioning/
     <yaml files>
 ```
 
-Next, let's look at how to provision a data source.
+Next, we'll look at how to provision a data source.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Provision a data source" %}}
 
-Each data source config file contains a _manifest_ that specifies the desired state of a set of data sources.
+Each data source configuration file contains a _manifest_ that specifies the desired state of a set of data sources. ((I'm a little lost. Is this specific to provisioning or is it in general? I infer from the next paragraph that it is probably provisioning-only, but you should probably clarify that up front.))
```

Comment:
Not sure I understand. The configuration file is Grafana-specific, specifically for provisioning data sources.

Reply:
I clarified the text. Please check once I push changes.